xcdat/spatial_avg.py

```diff
@@ -107,9 +107,9 @@ def avg(
 
         Get time series in Nino 3.4 domain:
 
-        >>> ts_n34 = ds.spatial.avg("tas", axis=["lat", "lon"],
+        >>> ts_n34 = ds.spatial.avg("ts", axis=["lat", "lon"],
         >>>     lat_bounds=(-5, 5),
-        >>>     lon_bounds=(-170, -120))["tas"]
+        >>>     lon_bounds=(-170, -120))["ts"]
```

Comment:
Hey @lee1043, is this the only example where `"tas"` should be `"ts"`?

I am referring to the description in #130.

Reply:
@tomvothecoder I think so. Thank you for following up.